examples/basic-examples/test/e2e/integration/custom/fonts.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Custom fonts should work
```

Comment:
```Feature: Custom font loader

  As a developer I can load custom fonts other than the default Vic font
      
  Scenario: Custom fonts are loaded in theme
    Given I visit the page "/"
    Then the h1 heading should have the fonts "Oswald, Arial, Helvetica, sans-serif"```

@tim-yao Do you mind writing the test like above? It will improve the readability of the test.